install_config/storage_examples/ceph_rbd_dynamic_example.adoc

```diff
@@ -0,0 +1,327 @@
+[[install-config-storage-examples-ceph-rbd-dynamic-example]]
+= Complete Example Using Ceph RBD for Dynamic Provisioning
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+This topic provides an end-to-end example of using an existing Ceph cluster as
+an {product-title} dynamic persistent store. It is assumed that a working Ceph cluster
+is already set up. If not, consult the
+link:https://access.redhat.com/products/red-hat-ceph-storage[Overview of Red Hat
+Ceph Storage].
+
+xref:../persistent_storage/persistent_storage_ceph_rbd.adoc#install-config-persistent-storage-persistent-storage-ceph-rbd[Persistent Storage
+Using Ceph Rados Block Device] provides an explanation of persistent volumes
+(PVs), persistent volume claims (PVCs), and using Ceph RBD as persistent
+storage.
+
+[NOTE]
+====
+All `oc ...` commands are executed on the {product-title} master host.
+====
+
+[[ceph-rbd-dynamic-example-installing-the-ceph-common-package]]
+== Installing the ceph-common Package
+
+The *ceph-common* library must be installed on *all schedulable* {product-title} nodes:
+
+[NOTE]
+====
+The {product-title} all-in-one host is not often used to run pod workloads and,
+thus, is not included as a schedulable node.
+====
+
+----
+# yum install -y ceph-common
+----
+
+[[ceph-rbd-dynamic-example-create-pool-for-dynamic-volumes]]
+== Create pool for dynamic volumes
+
+It is recommended to create a pool for your dynamic volumes to live.
+
+[NOTE]
+====
+Using the default pool of rbd is an option, but not recommended. 
+====
+
+From a Ceph admin or Mon node, create a new pool for dynamic volumes
```

Comment:
In general, I  see usage of *Mon* and *MON*. Is one more correct than the other? Not familiar with the term. We can just pick a way and keep consistent across the page